src/util/color/predefinedcolorsset.h

```diff
@@ -107,9 +109,28 @@ class PredefinedColorsSet final {
                 return color;
             }
         }
+        qWarning() << "No color matches id" << iId;
         return noColor;
     };
 
+    PredefinedColorPointer predefinedColorFromRgbColor(mixxx::RgbColor color) const {
```

Comment:
These utility functions don't access any instance-specific state and should be `static`

Reply:
They do access `allColors` and `noColor`. I probably could refactor the class to make everything static, but since we'll drop this whole class in #2520 anyway I'd rather not invest too much time.

Reply:
Correct. I didn't remember that both members are non-static.